package.json

```diff
@@ -29,12 +29,12 @@
     "url": "https://github.com/react-ui-org/react-ui"
   },
   "engines": {
-    "node": ">=20"
+    "node": ">=22"
   },
   "devEngines": {
     "runtime": {
       "name": "node",
-      "version": "22.x",
+      "version": "22.x || 24.x",
```

Comment:
For some reason, single version worked for matrix testing of both Node 20 and Node 22 (see `.github/workflows/build.yml`), but it does not work for Node 22 and Node 24. So I must entered both version (and not just latest one) to make it working. But I think this is correct approach.

Reply:
cc @adamkudrna @mbohal